app/admin/top-issues/components/AdminTopIssuesPage.js

```diff
@@ -54,33 +59,37 @@ export default function AdminTopIssuesPage(props) {
         </BlackButtonClient>
 
         {addNewIssue && (
-          <div>
-            <br />
-            <br />
+          <div  className="flex mt-4 items-center" >
+            <SVGIconChooser {...{
+              svgData,
+              setSvgData
+            }} />
             <TextField
+              className="mx-4"
               fullWidth
               primary
               label="Top Issue Name"
               value={topIssueName}
               onChange={(e) => setTopIssueName(e.target.value)}
             />
-            <br />
-            <br />
             <div className="text-right">
               <BlackButtonClient
                 disabled={topIssueName === ''}
                 onClick={handleCreate}
                 className="font-black"
               >
-                <strong>Save New Top Issue</strong>
+                <strong>Save</strong>
               </BlackButtonClient>
             </div>
           </div>
         )}
         <br />
         <br />
         <br />
-        <TopIssuesList {...props} />
+        <TopIssuesList {...{
+          topIssues,
+          setTopIssues
+        }} />
```

Comment:
I saw you pass props like this in your interview, and I was waiting for the chance to talk to you about it. Please follow our conventions:
```
 <TopIssuesList 
          topIssues={topIssues}
          setTopIssues={setTopIssues}
 />
```